android/java/res/layout/fragment_onboarding_network_selection.xml

```diff
@@ -110,7 +107,5 @@
             android:layout_alignParentStart="true"
             android:layout_alignParentEnd="true"
             android:text="@string/add_networks"
-            android:textColor="@color/wallet_tertiary_layout_text_color"
-            android:lineSpacingExtra="2sp"
-            android:textSize="12sp"/>
+            android:textColor="@color/wallet_tertiary_layout_text_color" />
```

Comment:
we should change the color here.

Reply:
Done in commit 👉 [`a85fb75` (#24063)](https://github.com/brave/brave-core/pull/24063/commits/a85fb75ba0e9f37da89ed68e35e9141a90eadbdf)